gamedev-iitk/tower-defense
# main menu and pause menu addedd

**Description:** (Have added main menu and pause menu)



**Steps to test:**
1. i have Created different scene "menu" file for main menu and linked it with "mainscene" file by changing build setting queue .
2. Have added settings but only with back button.
3. also in pause menu settings options we can change brightness of game .   
4. to Test pause menu we can check wave timer gets stopped while pause menu is active
